### ISSUE TITLE
REPL broken (first character) on Node 0.12

### ISSUE THREAD

**Issue by aseemk** [Sage/streamlinejs#283]
```
$ node --version
v0.12.7

$ _node --version
streamline v0.10.17

$ _node
_node> x = 5
Error: error streamlining source: :1:missing operand on line 1
  at Object.exports.transform (/path/to/streamline/lib/callbacks/transform.js:1752:10)
  at REPLServer.evaluate (/path/to/streamline/lib/compiler/repl.js:51:34)
  at bound (domain.js:254:14)
  at REPLServer.runBound [as eval] (domain.js:267:12)
  at REPLServer.<anonymous> (repl.js:279:12)
  at REPLServer.emit (events.js:107:17)
  at REPLServer.Interface._onLine (readline.js:214:10)
  at REPLServer.Interface._line (readline.js:553:8)
  at REPLServer.Interface._ttyWrite (readline.js:830:14)
  at ReadStream.onkeypress (readline.js:109:10)
  at ReadStream.emit (events.js:110:17)
  at readline.js:1175:14
  at Array.forEach (native)
  at emitKeys (readline.js:993:10)
  at ReadStream.onData (readline.js:910:14)
  at ReadStream.emit (events.js:107:17)
  at readableAddChunk (_stream_readable.js:163:16)
  at ReadStream.Readable.push (_stream_readable.js:126:10)
  at TTY.onread (net.js:538:20)

_node> ;x = 5
5

_node> x
Error: error streamlining source: :1:missing operand on line 1
  at Object.exports.transform (/path/to/streamline/lib/callbacks/transform.js:1752:10)
  at REPLServer.evaluate (/path/to/streamline/lib/compiler/repl.js:51:34)
  at bound (domain.js:254:14)
  at REPLServer.runBound [as eval] (domain.js:267:12)
  at REPLServer.<anonymous> (repl.js:279:12)
  at REPLServer.emit (events.js:107:17)
  at REPLServer.Interface._onLine (readline.js:214:10)
  at REPLServer.Interface._line (readline.js:553:8)
  at REPLServer.Interface._ttyWrite (readline.js:830:14)
  at ReadStream.onkeypress (readline.js:109:10)
  at ReadStream.emit (events.js:110:17)
  at readline.js:1175:14
  at Array.forEach (native)
  at emitKeys (readline.js:993:10)
  at ReadStream.onData (readline.js:910:14)
  at ReadStream.emit (events.js:107:17)
  at readableAddChunk (_stream_readable.js:163:16)
  at ReadStream.Readable.push (_stream_readable.js:126:10)
  at TTY.onread (net.js:538:20)

_node> ;x
5
```

Basically, the first character is stripped, it seems. (It can be anything, doesn't have to be a semi-colon.)

However, `.exit` works to exit, so not sure if that's always the case.

Known issue with Streamline 0.10? Fixed with Streamline 0.12?

**Comment by bjouhier** [comment]
Not known. I rarely use the REPL. Is it ok if I fix in 1.0 (the babel version)?

**Comment by aseemk** [comment]
Yep! Thanks Bruno.

**Comment by bjouhier** [comment]
Should be fixed in 1.0.2
